front/src/utils/allowedUserRoles.ts

```diff
@@ -0,0 +1,22 @@
+import useUserRoleCheck from 'common/authorization/hooks/useUserRoleCheck';
+import { REQUIRED_USER_ROLES_FOR } from 'common/authorization/roleBaseAccessControl';
+
+export default function useAllowedUserRoles() {
```

Comment:
Maybe this function should be defined in `common/authorization/hooks/useAllowedUserRoles`, instead of utils?